### ISSUE TITLE
questionis about epoch

### ISSUE THREAD

**Issue by fizzking** [amazon-science/earth-forecasting-transformer#23]

**Comment by gaozhihan** [comment]
This phenomenon does not necessarily mean that these training epochs end unexpectedly. One possible cause is that the outputs to the console somehow get flushed so that `\r` fails to get the original position to overwrite the previous logs. 
Could you please use [TensorBoard.dev](https://tensorboard.dev/) to check the log, especially the scalars that are logged with `self.log(..., on_step=True, ...)`, e.g., `train_loss`, to see if they got interrupted unexpectedly?

https://github.com/amazon-science/earth-forecasting-transformer/blob/093085ca0f7844c47d352fb55e32768b8f0bf07b/scripts/cuboid_transformer/nbody/README.md?plain=1#L8-L12

**Comment by fizzking** [comment]
> Yes, it will generate a tensorboard file and you may visualize the training loss via tensorboard.

I tried to input the code to upload the experiment log to tensorboard, but when I logged into my Google account, the page would not be displayed.
![image](https://user-images.githubusercontent.com/111410702/201005303-e948b528-55a0-40dc-a019-23a9c43d6f67.png)
![image](https://user-images.githubusercontent.com/111410702/201005321-8d6bdbec-d585-4acb-bebd-16bd31c39f8d.png)
![image](https://user-images.githubusercontent.com/111410702/201005327-9431e4ab-4faf-41b2-a7e6-3a78be0a19d0.png)
![image](https://user-images.githubusercontent.com/111410702/201005357-e79fcc44-7304-4911-9e04-8bd448b6c98e.png)

**Comment by gaozhihan** [comment]
The authorization of TensorBoard.dev should work correctly with appropriate network connection. 
If it still fails, you could try to use `tensorboard` to visualize your training log locally, following the official [instructions](https://www.tensorflow.org/tensorboard/get_started):
```
tensorboard --logdir YOUR_EXP_DIR
```

**Comment by gaozhihan** [comment]
Thanks for your issue. Please feel free to reopen it if you have any further questions.
